index.html

```diff
@@ -86,13 +86,23 @@
             button.onclick = weather;
         }
     </script>
+    <!-- Leaflet CSS for OpenStreetMap -->
+    <link rel = "stylesheet" href = "http://cdn.leafletjs.com/leaflet-0.7.3/leaflet.css" />
 </head>
 <body>
     <h1>Weather Forecast</h1>
     <label for="city">Enter City:</label>
     <input type="text" id ="city" placeholder="e.g. Seattle"></input>
 
     <button>Hey Yuri, how's the weather looking?</button>
-    <div id="weather"></div>
+    <div id="weather">
+        <div id="mapid" style="height: 300px; width: 500px; position: relative;"></div>
```

Comment:
added mapid div to attempt to re-size map but this does not work with leaflet.css styles coming in (at least that's what I think is the issue)